src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/models/RadPatch.java

```diff
@@ -39,7 +39,7 @@ public RadPatch(
 
     public record Revision(
             String id, String description, String base, String oid, List<String> refs,
-            List<Merge> merges, Instant timestamp, List<Discussion> discussions, List<String> reviews) { }
+            List<Merge> merges, Instant timestamp, List<Discussion> discussions, List<Object> reviews) { }
```

Comment:
I guess this is going to be implemented with a proper class when we add reviews?

Reply:
Yes